### ISSUE TITLE
New math rendering issue in mathlive

### ISSUE THREAD

**Issue by shivam0831** [arnog/mathlive#853]
Mathlive is having an issue with formatting sub superscript.
I have used this Latex code and it is not giving proper output as expected.
e^{\frac{x^2}{2}}

I have also tested the same code on the official website of mathlive i.e mathlive.io and the same result is giving.

Please help me resolve the issue!

I am using the following files to display static Latex code.
```
<!DOCTYPE html>
<html>
<head> 
<meta charset="UTF-8"> 
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mathlive@0.59.0/dist/mathlive-fonts.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mathlive@0.59.0/dist/mathlive-static.css">
</head> 
<body>
<div id="formula">
e^{\frac{x^2}{2}}
</div>
<script type="module">
import { renderMathInDocument } from 'https://cdn.jsdelivr.net/npm/mathlive@0.59.0/dist/mathlive.min.mjs';
renderMathInDocument();
</script>
</body>
</html>
```

One more thing I want to ask, How can I increase the font size of rendered Latex code?

**Comment by arnog** [comment]
To increase the font-size, set the CSS `font-size` property of the element you want to display. 

In your example:

```css
#formula {
   font-size: 32px
}
```

**Comment by arnog** [comment]
To increase the font-size, set the CSS `font-size` property of the element you want to display. 

In your example:

```css
#formula {
   font-size: 32px
}
```

**Comment by arnog** [comment]
Fixed in mathlive@0.60.0
